click-to-load/tracker_radar_reference.json

```diff
@@ -0,0 +1,36 @@
+{
+    "trackers": {
+        "tracker.test": {
```

Comment:
Since we don't have an entry for `tracker.net` in trackers as well, I would expect CTP blocking not to work for `tracker.net` (even though it's in the domains list in the CTP configuration). That's OK, but perhaps we should have a test case which shows that blocking isn't expected for domains which aren't considered tracking domains - even if there's a CTP entry which includes that domain?